### PR TITLE
Move TenantID to Cloud Credentials

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

**Review by cmurphy — COMMENTED** [review]
lgtm but I think needs to wait for tenant ID to be added to the cloud credential/machine driver

File: internal/aks/client.go [hunk 1]
```diff
@@ -94,11 +94,15 @@ func GetSecrets(secretsCache wranglerv1.SecretCache, spec *aksv1.AKSClusterConfi
 		return nil, fmt.Errorf("couldn't find secret [%s] in namespace [%s]", id, ns)
 	}
 
+	tenantIDBytes := secret.Data["azurecredentialConfig-tenantId"]
```
Comment (rmweir) [inline internal/aks/client.go:97]: You may want to add this to the example in the README where secret creation is demonstrated.

**Review by rmweir — COMMENTED** [review]
LGTM, aside from one comment about an addition to be made to README.

**Review by thedadams — APPROVED** [review]

File: internal/aks/client.go (hunk 1 above)
Comment (mjura) [reply internal/aks/client.go:97]: I have opened new PR https://github.com/rancher/aks-operator/pull/15

**Pull request merged**